paper-slider.html

```diff
@@ -441,23 +441,15 @@
     },
 
     _getClassNames: function() {
-      var classes = {};
-
-      classes.disabled = this.disabled;
-      classes.pin = this.pin;
-      classes.snaps = this.snaps;
-      classes.ring = this.immediateValue <= this.min;
-      classes.expand = this.expand;
-      classes.dragging = this.dragging;
-      classes.transiting = this.transiting;
-      classes.editable = this.editable;
-
-      return this._mergeClasses(classes);
-    },
-
-    _getProgressClass: function() {
       return this._mergeClasses({
```

Comment:
`_mergeClasses` is defined in paper-slider. I believe there's now a helper function in Polymer. I will check.